openemory/publication/models.py

```diff
@@ -1273,16 +1276,21 @@ class Publication(DigitalObject):
     PRESENTATION_CONTENT_MODEL = 'info:fedora/emory-control:PublishedPresentation-1.0'
 
     CONTENT_MODELS = []
+    #: collection this object belongs to
     collection = Relation(relsext.isMemberOfCollection)
+    #: OAI identifier
     oai_itemID = Relation(oai.itemID)
+    #: symplectic elements public url
+    public_url = Relation(SYMPLECTIC_MODEL_NS.hasPublicUrl)
```

Comment:
@alexBLR this is the piece that I'm not sure is adequately tested.  This relation definition _should_ be equivalent to the manual RDF triple handling that was being used before, but I don't know if existing test coverage confirms the triples or not.  If you think we need it, I can look at adding some unit tests to cover this relation specifically.